modules/viewing-the-log-analytics-to-detect-threats.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * integration/integrating-with-microsoft-sentinel-notifier.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-the-log-analytics-to-detect-threats_{context}"]
+= Viewing the log analytics to detect threats
+
+By creating a Microsoft sentinel integration, you can view the log analytics to detect threats.
+
+.Prerequisites
+
+* You have created a data collection rule, log analytics workspace, and service principal on Microsoft Azure.
+* You have configured a client secret or client certificate at the service principal for authentication.
+* You have created a log analytics schema. 
+** To create a schema, upload the following content to Microsoft sentinel: 
++
+.Example JSON
+[source,json]
+----
+{
+ "TimeGenerated": "2024-09-03T10:56:58.5010069Z", # <1>
+ "msg": {  # <2>
+   "id": "1abe30d1-fa3a-xxxx-xxxx-781f0a12228a" # <3>
```

Comment:
I've tried to articulate that the `msg` contains a JSON object of the alert, but only added the ID field.

Here is an example of the complete JSON send to sentinel: https://github.com/stackrox/stackrox/blob/master/central/notifiers/microsoftsentinel/data/example-alert.json

Imho we do not need to add the whole JSON here, it would be too costly to document it here. Do you have an idea how to represent it better?